### ISSUE TITLE
Failed to activate the linter-liferay package

### ISSUE THREAD

**Issue by jamieisboss** [mthadley/linter-liferay#9]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Ubuntu 16.04
**Thrown From**: [linter-liferay](https://github.com/mthadley/linter-liferay) package, v2.3.0
### Stack Trace

Failed to activate the linter-liferay package

```
At this.linter.provideLinter is not a function

TypeError: this.linter.provideLinter is not a function
  at Object.provideLinter (/home/jamie/.atom/packages/linter-liferay/lib/init.js:42:24)
  at Package.module.exports.Package.activateServices (/opt/atom/resources/app.asar/src/package.js:349:69)
  at Package.module.exports.Package.activateNow (/opt/atom/resources/app.asar/src/package.js:186:16)
  at /opt/atom/resources/app.asar/src/package.js:156:32
  at Package.module.exports.Package.measure (/opt/atom/resources/app.asar/src/package.js:92:15)
  at /opt/atom/resources/app.asar/src/package.js:149:26
  at Package.module.exports.Package.activate (/opt/atom/resources/app.asar/src/package.js:146:34)
  at PackageManager.module.exports.PackageManager.activatePackage (/opt/atom/resources/app.asar/src/package-manager.js:538:21)
  at /opt/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:389:60
  at exit (/opt/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:73:16)
  at triggerExitCallback (/opt/atom/resources/app.asar/src/buffered-process.js:215:47)
  at /opt/atom/resources/app.asar/src/buffered-process.js:229:18
  at Socket.<anonymous> (/opt/atom/resources/app.asar/src/buffered-process.js:100:18)
  at emitOne (events.js:82:20)
  at Socket.emit (events.js:169:7)
  at Pipe._onclose (net.js:469:12)

```
### Commands

```
     -1:55.3.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.seti-ui-no-icons.theme-firewatch-syntax.theme-one-dark-ui)
```
### Config

``` json
{
  "core": {
    "audioBeep": false,
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig"
      ],
      "source.json": [
        ".arcconfig",
        "BUCK.autodeps"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "disabledPackages": [
      "npm-docs",
      "platformio-aia",
      "platformio-ide",
      "platformio-ide-terminal",
      "tree-view",
      "linter",
      "Tern"
    ],
    "excludeVcsIgnoredPaths": false,
    "packagesWithKeymapsDisabled": [
      "platformio-ide"
    ],
    "projectHome": "/www",
    "themes": [
      "one-dark-ui",
      "firewatch-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
TinyRave, v2.2.1
activate-power-mode, v0.7.0
api-docs, v0.0.4
atom-notifier, v0.5.0
atom-react-native-autocomplete, v0.0.27
atom-react-native-css, v1.1.2
atomic-chrome, v0.3.0
autocomplete-clang, v0.9.4
autocomplete-jquery-mobile, v0.2.2
autocomplete-wordpress-hooks, v0.2.0
badass-react-snippets, v1.0.0
build, v0.65.0
busy, v0.5.0
console-log, v1.5.1
debug, v0.0.1
debug-service, v0.0.1
file-icons, v1.7.16
firewatch-syntax, v1.0.1
jQuery, v1.2.4
jquery-snippets, v11.0.0
jsonpp, v0.1.1
language-ansible, v0.1.6
language-arduino, v0.4.1
language-ini, v1.16.0
language-javascript-jsx, v0.3.7
language-multiverse, v1.0.3
language-sql-mysql, v0.3.1
lint, v0.0.1
linter-ansible-linting, v1.1.2
linter-bootlint, v1.0.2
linter-coffeelint, v1.1.2
linter-csslint, v1.3.2
linter-gcc, v0.6.14
linter-javac, v1.9.4
linter-js-yaml, v1.2.5
linter-jshint, v2.1.0
linter-json-lint, v0.1.1
linter-jsonlint, v1.3.0
linter-less, v2.5.0
linter-liferay, v2.3.0
linter-lsc, v2.3.0
linter-markdownlint, v1.0.1
linter-php, v1.2.0
linter-processing, v2.2.2
linter-pug-lint, v0.2.0
linter-pylint, v1.2.1
linter-ruby-lint, v1.2.0
linter-sass-lint, v1.6.1
linter-scss-lint, v3.0.4
linter-tidy, v2.1.1
linter-tslint, v0.10.0
linter-twig, v0.1.1
linter-xmllint, v1.3.5
livereload, v0.4.4
minimap, v4.24.7
multiple-directives, v1.0.3
nuclide, v0.148.0
php-debug, v0.2.3
pigments, v0.31.0
react-native-snippets, v0.3.0
relative-numbers, v0.5.1
syntax-theme-auto-changer, v0.9.4
terminal-plus, v0.14.5
tool-bar, v1.0.0
tool-bar-almighty, v0.6.1
wakatime, v6.0.10
wordpress-api, v1.2.2
wp-debug-watcher, v1.3.2

# Dev
No dev packages
```

**Comment by mthadley** [comment]
Seems to be happening when the package is upgraded to `2.3.0`, but everything looks fine after Atom is restarted. I'll investigate.

**Edit**: I also published a small change (`2.3.1`), but didn't notice any errors when upgrading.

**Comment by mthadley** [comment]
Closing this, however if there are still any issues, feel free to let me know. Thanks!
